pkg/controller/externalnode/controller.go

```diff
@@ -316,11 +319,7 @@ func (c *ExternalNodeController) deleteExternalNode(namespace string, name strin
 		return nil
 	}
 	en := obj.(*v1alpha1.ExternalNode)
-	eeName := externalnode.GenExternalEntityName(en)
```

Comment:
why the error is handled in addExternalNode but not in deleteExternalNode?

Reply:
I used to think that it can not occur for deletion case if the ExternalNode has been created. But you are right, once it is the case for or creation, then deletion should be the same. Updated it.